src/otx/algo/classification/backbones/efficientnet.py

```diff
@@ -425,17 +435,15 @@ def __init__(
         pooling_type: str | None = "avg",
         bn_eval: bool = False,
         bn_frozen: bool = False,
-        IN_first: bool = False,
-        IN_conv1: bool = False,
+        instance_norm_first: bool = False,
         pretrained: bool = False,
         **kwargs,
     ):
```

Comment:
Docstring and arguments are not aligned, it seems to be needed to be updated.
(I think there are other parts to be updated?)

Reply:
Is it okay to merge this PR first and fix this later since CI is not working properly today? (I'll fix this before CF.)

Reply:
Thanks! Done - https://github.com/openvinotoolkit/training_extensions/pull/3667/commits/eabd2936ee1820019da7d627d5a386317daca82b